src/mitochondria/transactions.ts

```diff
@@ -0,0 +1,29 @@
+import { TransactionType } from '../declarations/transaction';
+import { IAuthState } from '../store/reducers/auth';
+import {post} from './';
+
+/**
+ * Interface for body of a post request (Add Transaction) of the API
+ */
+export interface INewTransaction {
+  company_id: number;
+  money: number;
+  description: string;
+  date: string;  // ISO 8601 format: 'YYYY-MM-DD'
+  type: TransactionType;
+  notes?: string;
+}
+
+/**
+ * This function call the calls the @function post function to create the Transaction in the database.
+ * @param transaction The transaction we want to post to the API
+ * @param authState The authentication state for authenticating with the API
+ */
+export const createTransaction = async (transaction: INewTransaction, authState: IAuthState) => {
+  try {
+    const response = await post('/transaction', transaction, authState);
+    return response;
+  } catch (e) {
+    throw e;
+  }
+};
```

Comment:
We should discuss whether or not this kind of specialization is necessary.

Reply:
If you're thinking about the error handling, it is not in this case as catching and directly throwing an error is basically the same as doing nothing. The main difference is that it is (potentially) harder to see the true origin of the throw, as in this case it will be shown as coming from this function instead of its true source.